pyomo/solvers/tests/test_mindtpy3.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for the MINDT solver plugin."""
```

Comment:
Why is this called test_mindtpy3? Are there other tests? You can have multiple test functions in the same file/class.